src/main/groovy/com/deploygate/gradle/plugins/dsl/DeployGateExtension.groovy

```diff
@@ -27,9 +28,17 @@ class DeployGateExtension implements ExtensionSyntax {
     @Nonnull
     private final NamedDomainObjectContainer<NamedDeployment> deployments
 
-    DeployGateExtension(@Nonnull Project project, @Nonnull NamedDomainObjectContainer<NamedDeployment> deployments) {
+    DeployGateExtension(@Nonnull Project project, @Nonnull NamedDomainObjectContainer<NamedDeployment> deployments, @Nonnull CliCredentialStore credentialStore) {
         this.project = project
         this.deployments = deployments
+
+        this.appOwnerName = [System.getenv(DeployGatePlugin.ENV_NAME_APP_OWNER_NAME), System.getenv(DeployGatePlugin.ENV_NAME_APP_OWNER_NAME_V1), credentialStore.name].find {
```

Comment:
From now, the extension class sets the default values when initialized. This code block is transferred from the previous LoginTask.

Even if these values are filled in this constructor, the values through user's DSL will override them and it's the same behaviour to the previous code.